Core.IntegrationTests/Steps/ReleaseVersionAndMoveIssuesSubStepTests.cs

```diff
@@ -184,24 +184,24 @@ public void Execute_WithoutConfirmationToContinue_DoesNotMoveOpenIssues ()
     var testIssue3 = JiraTestUtility.AddTestIssueToVersion("Test closed Issue 3", true, c_jiraProjectKey, _restClient, initialVersion);
     var testIssue4 = JiraTestUtility.AddTestIssueToVersion("Test closed Issue 4", true, c_jiraProjectKey, _restClient, initialVersion);
 
-    var jiraSubStep = new ReleaseVersionAndMoveIssuesSubStep(
-        _console,
+    var jiraSubStep = new ReleaseVersionAndMoveIssuesSubStep(_console,
```

Comment:
```suggestion
    var jiraSubStep = new ReleaseVersionAndMoveIssuesSubStep(
        _console,
```
Not sure how many of those I missed in the rest of the changeset. pelase check.